Expecta/EXPDifference.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2020 Lightricks. All rights reserved.
+// Created by Maxim Grabarnik.
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// Protocol for computing difference between objects.
+@protocol EXPDifference <NSObject>
+
+/// Returns an array of strings describing the difference between the receiver and the given object.
+/// When the receiver and the given object are equal empty array is returned.
+///
+/// @note per convention the receiver is to be treated as the "expected",
+/// while the argument object is the "actual".
+- (NSArray<NSString *> *)differenceFrom:(id)object;
```

Comment:
do you guarantee anything about `object`? is it of the same class?
Maybe document some examples of the expected return value?

Reply:
done